src/common/transfer-position-modal/index.tsx

```diff
@@ -60,6 +62,7 @@ const TransferPositionModal = ({ position, open, onCancel }: TransferPositionMod
           </Typography>
         ),
       });
+      trackEvent('DCA - Transfer position submitting');
```

Comment:
Would be interesting to understand when was it clicked / opened, not only when submitted